scripts/process_jartic_full.py

```
@@ -90,14 +94,17 @@ def main():
                        help='Number of parallel workers (default: auto-detect)')
     parser.add_argument('--input-file', '-i', type=str, default=None,
                        help='Specific input CSV file to process (overrides date-based selection)')
+    parser.add_argument('--data-dir', '-d', type=str, default=None,
+                       help='Custom data directory path (e.g., external SSD path)')
     
     args = parser.parse_args()
     
     start_date = datetime.strptime(args.start, '%Y-%m-%d')
```

Ruff (0.12.2)

Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)